### ISSUE TITLE
Details Views Chip/ChipGroup should use PF Labels

### ISSUE THREAD

**Issue by AlexSCorey** [ansible/awx#5977]
##### ISSUE TYPE

- Bug Report

##### SUMMARY
Currently in many details views we show chip and/or chip group to show things like credentials, organizations, and and other items that should link out to other resources.  We need to switch out those chips for PF Labels that link out to the resource they represent.  Currently patternfly does not have the concept for Label Groups, but we might be able to use the inline List.  The developer who picks up this issue will have to audit the views to see whether they need to be updated.  

Click the link below....all the blue Labels should be linkable labels.
https://tower-mockups.testing.ansible.com/patternfly/templates/templates-detail/

**Comment by marshmalien** [comment]
related pf-react issues: 
https://github.com/patternfly/patternfly-react/issues/3829 
https://github.com/patternfly/patternfly-react/issues/4084

**Comment by unlikelyzero** [comment]
@marshmalien this is still blocked on those issues, right?

**Comment by marshmalien** [comment]
Yes, PatternFly hasn't released a LabelGroup component yet. The component is currently in the design phase.
https://github.com/patternfly/patternfly-design/issues/845

**Comment by mabashian** [comment]
I don't think this is blocked by PF anymore
